extensions/adobe/experience/target/classification.schema.json

```diff
@@ -0,0 +1,97 @@
+{
+    "meta:license": [
+        "Copyright 2023 Adobe Systems Incorporated. All rights reserved.",
+        "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+        "you may not use this file except in compliance with the License. You may obtain a copy",
+        "of the License at https://creativecommons.org/licenses/by/4.0/"
+    ],
+    "$id": "https://ns.adobe.com/experience/target/classification",
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "title": "Adobe Target Classification Fields",
+    "type": "object",
+    "description": "A set of meta-data fields that will classify Target activities and experiences.",
+    "definitions": {
+        "classification": {
+            "properties": {
+                "xdm:key": {
```

Comment:
Do you plan to put anything in the _id field? Will you duplicate the key or will you insert a uuid?

Reply:
Removing Key, will add the correlationId as _id.